drivers/sensor/lis2dw12/Kconfig

```diff
@@ -59,4 +59,16 @@ config LIS2DW12_TAP
 
 endif # LIS2DW12_TRIGGER
 
+config LIS2DW12_THRESHOLD
+	bool "Wakeup threshold trigger (via interrupt)"
+	help
+	  Enable the wakeup threshold trigger feature.
+	  The wake-up interrupt signal is generated if a certain number of
+	  consecutive data exceed the configured threshold (config in DT).
+	  The threshold is applied to both positive and negative data: for
+	  a wake-up interrupt generation at least one of the three axes must
+	  be bigger than the threshold. See ST AN5038 for more details about
+	  the feature.
+
+
```

Comment:
Please reconsider to have also this one in DT, or convince me that is not needed.

Reply:
Hi @avisconti .
The Kconfig choice here is about the firmware feature: code compiled in or not,
not about enabling/configuring this feature for a specific LIS2DW12 instance.

My board also has two LIS2DW12's on it, so I can still choose to only configure one in threshold mode. The user still needs to call `lis2dw12_attr_set_thresh()` on their specific instance to configure the feature.

It was mainly done from a code-size perspective, hope this is acceptable.

Reply:
> Hi @avisconti . The Kconfig choice here is about the firmware feature: code compiled in or not, not about enabling/configuring this feature for a specific LIS2DW12 instance.
> 
> My board also has two LIS2DW12's on it, so I can still choose to only configure one in threshold mode. The user still needs to call `lis2dw12_attr_set_thresh()` on their specific instance to configure the feature.
> 
> It was mainly done from a code-size perspective, hope this is acceptable.

Oh yes, sorry. It is called at runtime.
My bad...